torch/testing/_internal/common_modules.py

```diff
@@ -2524,6 +2524,23 @@ def module_error_inputs_torch_nn_LSTMCell(module_info, device, dtype, requires_g
     return samples
 
 
+def module_error_inputs_torch_nn_RNN_GRU(module_info, device, dtype, requires_grad, training, **kwargs):
+    make_input = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    samples = [
+        ErrorModuleInput(
+            ModuleInput(constructor_input=FunctionInput(10, 0, 1)),
+            error_on=ModuleErrorEnum.FORWARD_ERROR,
+            error_type=RuntimeError,
```

Comment:
oops, I think CI is failing because this should be ValueError

Reply:
Yes, thank you for your suggestion, i have fixed it.